test/webhook-proxy.test.ts

```diff
@@ -55,7 +62,7 @@ describe('webhook-proxy', () => {
 
     proxy = createWebhookProxy({url, logger: log})
 
-    proxy.on('error', err => {
+    proxy.on('error', (err: any) => {
```

Comment:
Can anyone point me to where this comes from?
It doesn't seem to exist in the code for `smee-client` and isn't a property of `EventSource`

Reply:
It seems as though what the author of this test was wanting to do was use `EventSource.addEventHandler` if you go by the function usage